packages/taquito-core/src/error/common.ts

```diff
@@ -183,3 +184,22 @@ export class ProhibitedActionError extends PermissionDeniedError {
     super(message);
   }
 }
+
+// Unsure the need to extend from high category ReponseError where there will only this error as child
+/**
+ *  @category Error
+ *  @description This error will be thrown when the endpoint returns an HTTP error to the client
+ */
+export class HttpResponseError extends Error {
```

Comment:
could this extend a parent?

Reply:
Will create httpError and extends from it